index.d.ts

```diff
@@ -0,0 +1,78 @@
+import { Action, StoreEnhancer, Store } from 'redux';
+
+interface StoreCreator {
```

Comment:
should all of these interfaces and types be exported? they are in redux's typings file. https://github.com/reactjs/redux/blob/master/index.d.ts

Reply:
👌  yeah seems good to just emulate what they're doing in the redux declaration